src/WorkOS.net/Services/DirectorySync/Entities/User.cs

```diff
@@ -86,6 +87,15 @@ public class User
         [JsonProperty("custom_attributes")]
         public Dictionary<string, object> CustomAttributes { get; set; }
 
+        /// <summary>
+        /// Grab User's primary email.
+        /// </summary>
+        /// <returns>The primary email of user.</returns>
+        public Email GetPrimaryEmail()
```

Comment:
We could use a getter here instead of making this a method:

```cs
public Email PrimaryEmail()
{
    get { return this.Emails.First(email => email.Primary == true) }
}
```